examples/tree_1d_dgsem/elixir_shallowwater_multilayer_beach.jl

```diff
@@ -0,0 +1,125 @@
+
+using OrdinaryDiffEq
+using Trixi
+using TrixiShallowWater
+
+###############################################################################
+# Semidiscretization of the shallow water equations
+
+equations = ShallowWaterMultiLayerEquations1D(gravity_constant = 9.812, rhos = 1.0)
```

Comment:
For clarity I would add a comment that passing a single value in `rhos` means this is a single layer, i.e., the standard shallow water equations.

Reply:
Done in 3d6d89a9381cd544f4ee470550c38ee47df42bf1